### DISCUSSION TITLE
how to move whole tree to another tree?

### DISCUSSION THREAD

**Discussion by zzzz465** [dendronhq/dendron#3454] [Q&A] [answered]
I searched discussions and issues before and couldn't find any... so.
let's assume we have tree-structured notes like the one below.

```
my-vault
 ┣ etc
 ┃ ┗ scala
 ┗ lang
 ┃ ┣ go
 ┃ ┗ ts
```

and each `/etc/scala`, `/lang/go`, `/lang/ts` have multiple nested sub notes.
how can I move "whole" `/etc/scala/**/*` notes to /lang/scala ?

**Comment by Harshita-mindfire** [comment, marked as answer]
You can use the [Dendron: Refactor Hierarchy](https://wiki.dendron.so/notes/9zwkp44wnlaa8p8dpt4w8tq/) command. It excepts a match text and replacement text and displays you a preview of all the notes that will be affected by the change.
In this particular case, the match text could be `etc.scala` and replacement `lang.scala`.
